airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_xcom.py

```diff
@@ -201,7 +201,7 @@ def test_custom_xcom_deserialize(
         self._create_xcom(TEST_XCOM_KEY, TEST_XCOM_VALUE, backend=XCom)
 
         url = f"/api/v2/dags/{TEST_DAG_ID}/dagRuns/{run_id}/taskInstances/{TEST_TASK_ID}/xcomEntries/{TEST_XCOM_KEY}"
-        with mock.patch("airflow.sdk.execution_time.xcom.XCom", XCom):
+        with mock.patch("airflow.sdk.bases.xcom.XCom", XCom):
```

Comment:
So this one I'm not sure about.

Having BaseXCom be in bases is fine, but I think the "resolved" backend shouldn't be in there and should remain somewhere in execution_time package

Reply:
I agree with this. It doesnt seem right because the XCom is resolved at execution time